lib/bulk_insert/worker.rb

```diff
@@ -130,6 +139,14 @@ def insert_ignore
       end
     end
 
+    def primary_key_return_statement
+      if adapter_name =~ /\APostgreSQL/i
+        ' RETURNING id'
```

Comment:
`id` is just the default primary key. Consider adding `primary_key` as an argument to the constructor for `BulkInsert::Worker` [and assign it in `BulkInsert::ClassMethods` L#9](https://github.com/procore/bulk_insert/blob/master/lib/bulk_insert.rb#L9) 

```ruby
worker = BulkInsert::Worker.new(connection, table_name, primary_key,  columns, set_size, ignore, update_duplicates)
```